### PR TITLE
Feature GE basic search

### PR DESCRIPTION
**Pull request by mihajlo-jaric**

### CODE REVIEW

File: WPFHospitalEditor/Building.xaml.cs [hunk 1]
```diff
@@ -44,12 +44,19 @@ private void populateBuildingFloors(List<MapObject> allBuildingObjects)
             }
         }
               
-        private String findFloor(MapObject mapObjectIteration)
+        public static String findFloor(MapObject mapObjectIteration)
         {
             String[] firstSplit = mapObjectIteration.Description.Split("&");
```
Comment (FmasterofU) [inline WPFHospitalEditor/Building.xaml.cs:49]: bad naming, try to find some kind of a meaningful variable name.
IMO even a one-liner would be better than a variable name like this

File: WPFHospitalEditor/Building.xaml.cs [hunk 2]
```diff
@@ -44,12 +44,19 @@ private void populateBuildingFloors(List<MapObject> allBuildingObjects)
...
             String[] floor = firstSplit[0].Split("-");
             return floor[1];
-        }      
+        }
+
+        public static String findBuilding(MapObject mapObjectIteration)
+        {
+            String[] firstSplit = mapObjectIteration.Description.Split("&");
```
Comment (FmasterofU) [inline WPFHospitalEditor/Building.xaml.cs:56]: same as previous comment(variable naming)

File: WPFHospitalEditor/SearchResultDialog.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+        private void addAdvancedSearchButton()
+        {
+            Button advancedSearch = new Button();
+
+            adjustAdvancedSearchButtonProperties(advancedSearch);
+
+            advanceSearchButtons.Add(advancedSearch);
+
+            //advancedSearch.Click += Advanced_Search_Click(mapObject, i);
```
Comment (FmasterofU) [inline WPFHospitalEditor/SearchResultDialog.xaml.cs:140]: unless it's absolutely necessary, avoid leaving comments, especially commented code

File: WPFHospitalEditor/SearchResultDialog.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+            HospitalMap.searchResult.Clear();
+            Close();
+        }
+
+        /*private void Advanced_Search_Click(MapObject mapObject, int i)
+        {
+            button.Click += new EventHandler(
+                (sender, e) => button_Click(sender, e, grid, i)
+            );
+        }*/
```
Comment (FmasterofU) [inline WPFHospitalEditor/SearchResultDialog.xaml.cs:217]: same as before, commented code, and comments in general, should only be there if it's absolutely necessary

**Review by FmasterofU — CHANGES_REQUESTED** [review]
variable naming and comments

File: WPFHospitalEditor/SearchResultDialog.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+                        label.Content = Building.findBuilding(mapObject);
+                        label.HorizontalAlignment = HorizontalAlignment.Center;
+                        label.VerticalAlignment = VerticalAlignment.Center;
+                    }
+                    break;
+                case 3:
+                    {
+                        label.Content = Building.findFloor(mapObject);
+                        label.HorizontalAlignment = HorizontalAlignment.Center;
+                        label.VerticalAlignment = VerticalAlignment.Center;
```
Comment (rammba) [inline WPFHospitalEditor/SearchResultDialog.xaml.cs:105]: Move these after switch because same lines are repeating in all cases.

**Comment by rammba** [comment]
[(0) PR Feature GE basic search](https://trello.com/c/A5Mm1MZM/83-0-pr-feature-ge-basic-search)

File: WPFHospitalEditor/SearchResultDialog.xaml.cs (hunk 3 above)
Comment (mihajlo-jaric) [reply WPFHospitalEditor/SearchResultDialog.xaml.cs]: Those comments are related to next feature, but I will delete them.

File: WPFHospitalEditor/SearchResultDialog.xaml.cs (hunk 4 above)
Comment (FmasterofU) [reply WPFHospitalEditor/SearchResultDialog.xaml.cs]: explained, next feature

**Review by FmasterofU — APPROVED** [review]

**Review by rammba — APPROVED** [review]

**Pull request merged**